src/containers/SearchPage/components/form/SearchContentForm.tsx

```diff
@@ -18,21 +18,59 @@ import ObjectSelector from '../../../../components/ObjectSelector';
 import SearchTagGroup from './SearchTagGroup';
 import ArticleStatuses from '../../../../util/constants/index';
 import { fetchAuth0Editors } from '../../../../modules/auth0/auth0Api';
-import { searchFormClasses } from './SearchForm';
-import { LocationShape } from '../../../../shapes';
+import { searchFormClasses, SearchParams } from './SearchForm';
+import { LocationShape, SearchParamsShape } from '../../../../shapes';
 import { DRAFT_WRITE_SCOPE } from '../../../../constants';
+import { SubjectType } from '../../../../interfaces';
 
-const emptySearchState = {
+const emptySearchState: SearchState = {
   query: '',
   subjects: '',
   resourceTypes: '',
   status: '',
   users: '',
-  lang: '',
+  language: '',
```

Comment:
Denne heiter lang av en grunn. Det er fordi det ikkje finnes language-dropdown for innhold, og ved å kalle denne for lang så dukker heller ikkje språket som brukes her opp i lista med valgte søkefilter under søkefeltet.

Reply:
Ser at det dukker opp en tom Innhold-knapp i lista over valgte filter.

Reply:
Riktig, prøver å fikse det :+1: